migrate/live-migration/live-migration-from-timescaledb.md

```diff
@@ -82,44 +82,62 @@ which will impact the consistency of your data post migration.
 
 <LiveMigrationDockerSubcommand />
 
-`migrate` will utilize the snapshot created in the previous step and migrate existing data to the
-target. Then, it will stream live transactions from the source to the target. During this process,
-it will display the lag between the source and target databases in terms of WAL offset size.
+The `migrate` command utilizes the snapshot created in the previous step and
+migrates existing data to the target. Then, it streams live transactions from
```

Comment:
It then streams live ... 

During this process, you see the replay process status:

Reply:
done